src/util/notifications_util.js

```diff
@@ -422,7 +422,7 @@ function compose_notification_req(req, res, bucket, notif_conf) {
     let eTag = res.getHeader('ETag');
     //eslint-disable-next-line
     if (eTag && eTag.startsWith('\"') && eTag.endsWith('\"')) {
```

Comment:
It starts with '\"' ?  shouldn't it be '"\' ?    For example: "\test-etag\"
Please keep one example in comment to avoid confusion for code reader.

Reply:
The only option currently for quotation is with double quote marks, "".
That is the only test performed in 424.
I have not seen an etag quoted with single quotes '' yet, so I didn't test for it.